book/src/feature/lang_tuple.md

```diff
@@ -0,0 +1,14 @@
+# Tuples
+
+Introduced in Dart 3, [records](https://dart.dev/language/records) provide the equivalent of Rust's tuples.
```

Comment:
Hey I realize https://github.com/fzyzcjy/flutter_rust_bridge/blob/b7ec2dabe9f6582c3d544f0e1239d3cba18b3dfb/frb_dart/pubspec.yaml#L6 we already require Dart 3 in our lib.

Thus, maybe we do not need to introduce this flag, and can make it the default?

(This can be done only if this flag will not introduce any breaking change)

Reply:
It is a breaking change, since class modifiers like `sealed` were only introduced in Dart 3 and would cause a syntax error on previous versions. But if we're already on 3, it makes good sense to make it the default.